packages/edit-post/src/index.js

```diff
@@ -91,6 +93,7 @@ export function initializeEditor(
 			enableFSEBlocks: settings.__unstableEnableFullSiteEditingBlocks,
 		} );
 	}
+	registerDefaultActions();
```

Comment:
I couldn't find a better place to call this. I kind of maps what we do for blocks.